S/Spasm/build_tarballs.jl

```diff
@@ -0,0 +1,73 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Spasm"
```

Comment:
Why name it "Spasm" and not "SpaSM" as the source specifies, https://github.com/cbouilla/spasm ?

Reply:
Good suggestion, will do.